core/math/a_star.h

```diff
@@ -50,8 +50,8 @@ class AStar3D : public RefCounted {
 		real_t weight_scale = 0;
 		bool enabled = false;
 
-		AHashMap<int64_t, Point *> neighbors = 4u;
-		AHashMap<int64_t, Point *> unlinked_neighbours = 4u;
+		AHashMap<int64_t, Point *> neighbors{ 4u };
+		AHashMap<int64_t, Point *> unlinked_neighbours{ 4u };
```

Comment:
Taken from the C++ usage guidelines, we prefer `A = {}` over `A{}` where possible. The `u` suffix is a whole other can of worms, especially considering I don't know if there's actually consensus on that end, but it's superfluous here because we're passing a literal.
```suggestion
		AHashMap<int64_t, Point *> neighbors = { 4 };
		AHashMap<int64_t, Point *> unlinked_neighbours = { 4 };
```